### PR TITLE
Update matrix-org-eslint-plugin and tighten max warning limit

### PR DESCRIPTION
**Pull request by t3chguy**
Whilst it is down, make the most of it!

Notes: none

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -7331,8 +7331,8 @@ matrix-mock-request@^1.2.3:
     counterpart "^0.18.6"
     diff-dom "^4.2.2"
     diff-match-patch "^1.0.5"
-    emojibase-data "^6.2.0"
-    emojibase-regex "^5.1.3"
+    emojibase-data "^5.1.1"
+    emojibase-regex "^4.1.1"
```
Comment (turt2live) [inline yarn.lock:7335]: unrelated?

Comment (t3chguy) [reply yarn.lock:7335]: Indeed, what on earth are you doing yarn

Comment (turt2live) [reply yarn.lock:7335]: being yarn

**Review by turt2live — APPROVED** [review]
